src/components/Input/Input.module.scss

```diff
@@ -0,0 +1,36 @@
+@import "../../assets/variables.scss";
+
+label {
```

Comment:
(nice-to-have) Are these styles applied globally for the file that they're imported in? Alternatively these could be changed to classes and imported in the component

(nice-to-have-x2) I haven't seen `px`s, `em`s and `rem`s in a single file for quite some time 😅 Maybe we could unify the units?